app/src/androidTest/java/org/mozilla/focus/helpers/TestHelper.kt

```diff
@@ -162,6 +171,70 @@ object TestHelper {
         }
     }
 
+    fun verifyDownloadedFileOnStorage(fileName: String) {
+        val context = InstrumentationRegistry.getInstrumentation().targetContext
+        val resolver = context.contentResolver
+        val fileUri = queryDownloadMediaStore(fileName)
+
+        val fileExists: Boolean = fileUri?.let {
+            resolver.openInputStream(fileUri).use {
+                it != null
+            }
+        } ?: false
+
+        assertTrue(fileExists)
+    }
+
+    /**
+     * Check the "Downloads" public directory for [fileName] and returns an URI to it.
+     * May be `null` if there is no file with that name in "Downloads".
+     */
+    @TargetApi(Build.VERSION_CODES.Q)
+    @VisibleForTesting
```

Comment:
This line is not needed.
It would allow to relax it's visibility just for when it is being tested - we used this for methods that were to be unit tested. Here it's not the case.

Reply:
Sure, consider it done!
Many thanks! 🙏🏽